### DISCUSSION TITLE
蓝盾构建history页面加重试次数字段

### DISCUSSION THREAD

**Discussion by leozlliang** [Tencent/bk-ci#5026] [Ideas]
<img width="398" alt="ce3b0115-136a-41b0-a7fc-26c87b571cb6" src="https://user-images.githubusercontent.com/9716704/130723341-52c3db5f-7cd9-442a-bc9d-6f39a2cd0a4c.png">
问题：蓝盾构建如果目前发生过重试，那么用户很难直观感知到某个构建有重试过。只能通过组件上的重试次数判断。

解决方案：蓝盾构建history页面加重试次数字段，让用户直观查找到。

**Comment by iccengan** [comment]
已在下半年规划内，我们会对构建详情页面进行重构。将支持重试字段，并查看对应重试次数的流水线编排。
https://github.com/Tencent/bk-ci/issues/2981
